app/services/school_search.rb

```diff
@@ -8,6 +8,7 @@ class SchoolSearch
 
   def initialize(query: nil, limit: DEFAULT_LIMIT, lead_schools_only: false)
     @query = StripPunctuation.call(string: query)
+    @query = ReplaceAbbreviation.call(string: query)
```

Comment:
We got a double @ query here 👀

Reply:
oh hold on.. it's supposed to work like that! confused me for a mo.

Reply:
No, you're right. It should have been:

```ruby
query = StripPunctuation.call(string: query)
@query = ReplaceAbbreviation.call(string: query)
```

or maybe

```ruby
@query = ReplaceAbbreviation.call(string: StripPunctuation.call(string: query))
```